wire-ios/Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsPropertyToggleCellDescriptor.swift

```diff
@@ -83,7 +85,17 @@ final class SettingsPropertyToggleCellDescriptor: SettingsPropertyCellDescriptor
         }
 
         do {
-            try self.settingsProperty << SettingsPropertyValue(valueToSet)
+            try self.settingsProperty.set(newValue: SettingsPropertyValue(valueToSet), resultHandler: { result in
+                if case .failure = result {
+                    // Not idea but a workaround
```

Comment:
```suggestion
                    // Workaround: the toggle needs to be undone because of an async result in the logic code. 
                    // Problem is, the logic is so far removed from the UI that we can only communicate the error by passing a result handler.
```

Reply:
Fixed!